doc/user/zebra.rst

```diff
@@ -407,6 +407,14 @@ If no option is chosen, then the *Linux VRF* implementation as references in
 https://www.kernel.org/doc/Documentation/networking/vrf.txt will be mapped over
 the *Zebra* VRF. The routing table associated to that VRF is a Linux table
 identifier located in the same *Linux network namespace* where *Zebra* started.
+Please note when using the *Linux VRF* routing table it is expected that a
+default Kernel route will be installed that has a metric as outlined in the
+www.kernel.org doc above.  The Linux Kernel does table lookup via a combination
+of rule application of the rule table and then route lookup of the specified
+table.  If no route match is found then the next applicable rule is applied
+to find the next route table to use to look for a route match.  As such if
+your VRF table does not have a default blackhole route with a high metric
+VRF route lookup will leave the table specified the VRF, which is undesirable.
```

Comment:
"leave the table specified the VRF" - that's a little hard to parse? should it be "lookup processing will leave the VRF, and search the default VRF table", something like that?

Reply:
but it will not necessarily do that.  I cannot predict the order of rules that someone may have and I refuse to imply something that is not necessarily true.

Reply:
I meant that the sentence is missing at least a work in the phrase "specified the VRF". Should that be "specified by the VRF", or "that specified the VRF", or something else?

Reply:
fixed the sentence